docs/docs/20-concepts/02-mcp-gateway/03-obot-registry.md

```diff
@@ -0,0 +1,288 @@
+# Obot's Open Source Registry
+Each Obot instance connects to our open source registry by default. You can follow this guide to add your MCP server to [Obot’s registry](https://github.com/obot-platform/mcp-catalog). You’ll either submit **just a catalog entry**, or **a catalog entry + a repackaged image** (for stdio servers).
+
+---
+
+## Step 0 — Pick your path
+
+**Question:** What type of MCP server are you shipping?
```

Comment:
I think there are a few missing scenarios from this - what if they have a docker image thats stdio? What if they have a npx/uvx server thats streamable http?